contrib/test_cover.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-PKGS=$(go list ./... | grep -v '/simapp' | grep -v '/cli_test')
+PKGS=$(go list ./... | grep -v '/simapp')
```

Comment:
I'm not sure this script is used at all. It is mentioned in a Makefile's target but it does not seem to be used anymore.

Reply:
It's used by `test-cover`, but that isn't used by anything. Let's remove it in a subsequent PR.